h2o-docs/src/product/data-science/if.rst

```diff
@@ -71,6 +80,68 @@ Defining an Isolation Forest Model
   - ``label_encoder`` or ``LabelEncoder``:  Convert every enum into the integer of its index (for example, level 0 -> 0, level 1 -> 1, etc.)
   - ``sort_by_response`` or ``SortByResponse``: Reorders the levels by the mean response (for example, the level with lowest response -> 0, the level with second-lowest response -> 1, etc.). This is useful in GBM/DRF, for example, when you have more levels than ``nbins_cats``, and where the top level splits now have a chance at separating the data with a split. Note that this requires a specified response column.
 
+Simple Example
+~~~~~~~~~~~~~~
+
+Below is a simple example showing Isolation Forest from model training through prediction and predicted leaf node assignment. 
+
+.. example-code::
+   .. code-block:: r
+
+    library(h2o)
+    h2o.init()
+
+    # Import the prostate dataset
+    prostate.hex <- h2o.importFile(path = "https://raw.github.com/h2oai/h2o/master/smalldata/logreg/prostate.csv", 
+                                   destination_frame = "prostate.hex")
+
+    # Split dataset giving the training dataset 75% of the data
+    prostate.split <- h2o.splitFrame(data=prostate.hex, ratios=0.75)
+
+    # Create a training set from the 1st dataset in the split
+    train <- prostate.split[[1]]
+
+    # Create a testing set from the 2nd dataset in the split
+    test <- prostate.split[[2]]
+
+    # Build an Isolation forest model
+    model <- h2o.isolationForest(training_frame=train, 
```

Comment:
I think we can remove the option `score_each_iteration = TRUE`

Reply:
done!